<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDataDialog</class>
 <widget class="QDialog" name="AddDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加联系人</string>
  </property>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="NameEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="PhoneEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="CityEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>81</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Namelabel">
      <property name="text">
       <string>  姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Phonelabel">
      <property name="text">
       <string>  电话</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Citylabel">
      <property name="text">
       <string>  城市</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDataDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YesButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDataDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
